arm-network/2017-03-01/swagger/networkWatcher.json

```diff
@@ -1714,6 +1769,176 @@
+        },
+        "minLatencyInMs": {
+          "description": "Minimum latency in milliseconds.",
+          "type": "integer"
+        },
+        "maxLatencyInMs": {
+          "description": "Maximum latency in milliseconds.",
+          "type": "integer"
+        },
+        "probesSent": {
+          "description": "Total number of probes sent.",
+          "type": "integer"
+        },
+        "probesFailed": {
+          "description": "Number of failed probes.",
+          "type": "integer"
+        }
+      }
+    },
+    "ConnectivityHop": {
+      "description": "Information about a hop between the source and the destination.",
+      "properties": {
+        "type": {
+          "description": "The type of the hop.",
+          "type": "string"
+        },
+        "id": {
+          "description": "The ID of the hop.",
+          "type": "string"
+        },
+        "address": {
+          "description": "The IP address of the hop.",
+          "type": "string"
+        },
+        "resourceId": {
+          "description": "The ID of the resource corresponding to this hop.",
+          "type": "string"
+        },
+        "nextHopIds": {
+          "type": "array",
+          "description": "List of next hop identifiers.",
+          "items": {
+            "type": "string"
+          }
+        },
+        "issues": {
+          "type": "array",
+          "description": "List of issues.",
+          "items": {
+            "$ref": "#/definitions/ConnectivityIssue"
+          }
+        }
+      }
+    },
+    "ConnectivityIssue": {
+      "description": "Information about an issue encountered in the process of checking for connectivity.",
+      "properties": {
+        "origin": {
+          "type": "string",
+          "enum": [ "Local", "Inbound", "Outbound" ],
+          "x-ms-enum": {
+            "name": "Origin",
+            "modelAsString": true
+          },
+          "description": "The origin of the issue."
+        },
+        "severity": {
+          "type": "string",
+          "enum": [ "Error", "Warning" ],
+          "x-ms-enum": {
+            "name": "Severity",
+            "modelAsString": true
+          },
+          "description": "The severity of the issue."
+        },
+        "type": {
+          "type": "string",
+          "enum": [ "Unknown", "AgentStopped", "GuestFirewall", "DnsResolution", "SocketBind", "NetworkSecurityRule", "UserDefinedRoute", "PortThrottled", "Platform" ],
+          "x-ms-enum": {
+            "name": "IssueType",
+            "modelAsString": true
+          },
+          "description": "The type of issue."
+        },
+        "context": {
+          "type": "array",
+          "description": "Provides additional context on the issue.",
+          "items": {
+            "$ref": "#/definitions/IssueContext"
+          }
+        }
+      }
+    },
+    "IssueContext": {
+      "description": "A key-value pair that provides additional context on the issue.",
```

Comment:
Just a single key-value pair? I don't know your API so this is perfectly fine if it's correct, just seemed odd :-) So above, for property `context`, you send values like
```
[
   { "key1": "value1" },
   { "key2": "value2" },
   { "key3": "value3" }
]
```
? Just making sure